### PR TITLE
ELEMENTS-1449: fix running status checking for bulk actions

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

File: ui/widgets/nuxeo-operation-button.js [hunk 1]
```diff
@@ -328,19 +328,16 @@ import '../actions/nuxeo-action-button-styles.js';
...
-        return state !== 'ABORTED' && state !== 'COMPLETED';
-      }
-      return status === 'RUNNING';
+      const state = this._hasBulkStatus(status) ? (status.value && status.value.state) || status.state : status;
+      return state === 'RUNNING';
     }
 
     _isAborted(status) {
-      if (status && status['entity-type'] === 'bulkStatus') {
+      if (this._hasBulkStatus(status)) {
```
Comment (Gabez0r) [inline ui/widgets/nuxeo-operation-button.js]: Can't we do a one-liner like we did on the `_isRunning`?

**Review by Gabez0r — COMMENTED** [review]
LGTM

File: ui/widgets/nuxeo-operation-button.js (hunk 1 above)
Comment (richardsd) [reply ui/widgets/nuxeo-operation-button.js]: Sure, the logic is a bit different, but let me refactor this one too.

**Comment by richardsd** [comment]
Cross repo pipeline https://github.com/nuxeo/nuxeo-web-ui/runs/4562786521 🔵

File: ui/widgets/nuxeo-operation-button.js (hunk 1 above)
Comment (Gabez0r) [reply ui/widgets/nuxeo-operation-button.js]: Ah sorry, they seemed very similar. I'm ok with either, whatever is easier to read!

**Review by Gabez0r — APPROVED** [review]

**Review by mnixo — APPROVED** [review]

**Pull request merged**